cli/azd/internal/appdetect/testdata/dotnet/dotnettestapp.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
We should update project's Readme, like https://github.com/Azure/azure-dev/blob/main/templates/todo/projects/csharp-cosmos-sql/README.md?plain=1#L43

Reply:
done